heron/scheduler-core/src/java/com/twitter/heron/scheduler/utils/SchedulerConfigUtils.java

```diff
@@ -58,12 +58,15 @@ private static Config topologyConfigs(String topologyBinaryFile,
    * @return config, the command line config
    */
   private static Config commandLineConfigs(String cluster, String role,
-                                           String environ, Boolean verbose) {
+                                           String environ, Boolean verbose,
+                                           String autoHeal, String autoHealInterval) {
```

Comment:
What are the acceptable values for autoHeal and autoHealInterval and can these be Duration?